config/common-dualstack.cfg

```diff
@@ -2,3 +2,5 @@
 --resource_prefix=psm-ds
 --enable_dualstack
 --noenable_workload_identity
+--compute_api_version=v1beta
```

Comment:
Let's add a todo comment to remove this flag once everything is available in compute v1

Reply:
done